### PR TITLE
Move config variables to config package

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -97,3 +97,107 @@ var (
...
+var (
+	RegionCacheBTreeDegree = 32
+	RegionCacheTTL         = time.Minute * 10
+)
+
+// RawKV configurations.
+var (
+	// MaxRawKVScanLimit is the maximum scan limit for rawkv Scan.
+	MaxRawKVScanLimit = 10240
+	// rawBatchPutSize is the maximum size limit for rawkv each batch put request.
```
Comment (rleungx) [inline config/config.go]:
```suggestion
	// RawBatchPutSize is the maximum size limit for rawkv each batch put request.
```

File: config/config.go [hunk 2]
```diff
@@ -97,3 +97,107 @@ var (
...
+	RawBatchPutSize = 16 * 1024
+	// rawBatchPairCount is the maximum limit for rawkv each batch get/delete request.
```
Comment (rleungx) [inline config/config.go]:
```suggestion
	// RawBatchPairCount is the maximum limit for rawkv each batch get/delete request.
```

**Comment by siddontang** [comment]
why not passing the config var directly?

**Comment by disksing** [comment]
@siddontang  Plan to do it in a separated PR.

**Pull request merged**